### DISCUSSION TITLE
Get token's lighting condition

### DISCUSSION THREAD

**Discussion by Frstrm** [dev7355608/perfect-vision#312] [Ideas]
Is there a method/function that when getting/selecting a token or it's position, you can determine the light condition it's in ( dark, dim, bright )?

**Comment by dev7355608** [comment]
I don't have a function for that in PV. The delimiter overlay (CTRL+H) can give you an idea how bright it is, but that might not be what you are looking for.

**Reply by Frstrm** [reply]
Thanks, yeah I was looking for a call that I could make that would return the brightness at a tokens location.  The delimiter is great for when doing manual determination for rolls.  ( i.e. player has a feat that sets that they can't be seen with darkvision when they are in darkness, etc. ).
